pyanaconda/modules/services/services.py

```diff
@@ -187,3 +188,19 @@ def set_setup_on_boot(self, value):
         self._setup_on_boot = value
         self.setup_on_boot_changed.emit()
         log.debug("Setup on boot is set to %s.", value)
+
+    def install_with_tasks(self, sysroot):
+        """Return the installation tasks of this module.
+
+        :param str sysroot: a path to the root of the installed system
+        :returns: list of object paths of installation tasks
+        """
+        tasks = [
+            ConfigureInitialSetupTask(sysroot=sysroot, setup_on_boot=self.setup_on_boot)
+        ]
+
+        paths = [
+            self.publish_task(SERVICES.namespace, task) for task in tasks
+        ]
```

Comment:
Do we really need `for` here for just one task?

Reply:
At the moment there is just a single task, but there should be more soon for the other kickstart commands handled by the services module - "skipx", "services", "xconfig"

Reply:
In that case it could be here fine.